assets/src/scripts/components/hydrograph/time-series.js

```diff
@@ -38,71 +36,6 @@ export const TEMPERATURE_PARAMETERS = {
 // Create a time formatting function from D3's timeFormat
 const formatTime = timeFormat('%c %Z');
 
-/**
- * Returns a selector that, for a given tsKey and period:
- * Selects all time series data. If period is null then, the currentDateRange is used
- * @param  {String} tsKey   Time-series key
- * @param {String} period or null date range using ISO-8601 duration;
- * @param  {Object} state   Redux state
- * @return {Object} - Keys are tsID, values are time-series data
- */
-export const getTimeSeriesForTsKey = memoize((tsKey, period) => createSelector(
```

Comment:
The selectors that were removed are needed to manage the url-params. That module is in the base scripts directory and I did not want it to depend on module that should only be used by hydrograph. We could probably stand some selector clean-up but that will need to wait.